samples/logging/prometheus-grafana-opentelemetry/Core_8/Endpoint/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    public static async Task Main()
+    {
+        #region enable-prometheus-exporter
+        var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddMeter("NServiceBus.Diagnostics")
+            .AddPrometheusExporter(opt =>
+            {
+                opt.StartHttpListener = true;
+                opt.HttpListenerPrefixes = new[]
+                {
+                    "http://localhost:9185",
+                    "http://192.168.0.114:9184"
+                };
+                opt.ScrapeEndpointPath = "/metrics";
+            })
+            .Build();
+
+        #endregion
+
+        var config = new EndpointConfiguration("Samples.OpenTelemetry.Metrics");
+        config.UseTransport<LearningTransport>();
+        config.UsePersistence<LearningPersistence>();
+        config.UseSerialization<XmlSerializer>();
+        config.MakeInstanceUniquelyAddressable("main");
```

Comment:
is this necessary?

Reply:
I wanted to elevate this on the dashboard, so yes.

Reply:
what does this do on the dashboard? The endpoint should never consume from this separate queue?

Reply:
This allows further filtering on the dashboard. Maybe irrelevant here as it is a simple example?

Reply:
Removed it from the sample, as it may not make sense here.